tests/VirtoCommerce.SubscriptionModule.Test/SubscriptionExportImportTests.cs

```diff
@@ -240,37 +202,23 @@ public async Task TestDataExport()
                 }
             }
 
-            // Assert
-            // NOTE: whitespace characters and indentation may vary, so comparing JSON as text will not work well.
-            //       To overcome it, we compare JSON tokens here.
-            var expectedJObject = JsonConvert.DeserializeObject<JObject>(expectedJson);
-            var actualJObject = JsonConvert.DeserializeObject<JObject>(actualJson);
-            Assert.True(JToken.DeepEquals(expectedJObject, actualJObject));
-        }
-
-        [Fact]
-        public async Task TestDataImport()
-        {
-            // Arrange
-            Subscription[] actualSubscriptions = null;
-            _subscriptionService.Setup(service => service.SaveSubscriptionsAsync(It.IsAny<Subscription[]>()))
-                .Callback<Subscription[]>(subscriptions => actualSubscriptions = subscriptions)
-                .Returns(Task.CompletedTask);
-
-            PaymentPlan[] actualPaymentPlans = null;
-            _paymentPlanService.Setup(service => service.SavePlansAsync(It.IsAny<PaymentPlan[]>()))
-                .Callback<PaymentPlan[]>(paymentPlans => actualPaymentPlans = paymentPlans)
-                .Returns(Task.CompletedTask);
 
-            // Act
-            using (var resourceStream = ReadEmbeddedResource("Resources.SerializedSubscriptionData.json"))
-            {
-                await _subscriptionExportImport.DoImportAsync(resourceStream, IgnoreProgressInfo, _cancellationToken.Object);
-            }
+            var importStream = GetStreamFromString(actualJson);
+            await _subscriptionExportImport.DoImportAsync(importStream, IgnoreProgressInfo, _cancellationToken.Object);
 
-            // Assert
+            //Asset
             Assert.Equal(TestSubscriptions, actualSubscriptions);
```

Comment:
You need to use another comparison here, which will compare all objects properties

Reply:
`TestSubscriptions.Should().BeEquivalentTo(actualSubscriptions);`

Reply:
updated